tests/capi/unittest_capi_service_agent_client.cc

```diff
@@ -1154,14 +1154,11 @@ TEST_F (MLServiceAgentTest, model_scenario)
 
   g_free (info_list);
 
-  /* delete the active model */
+  /* failed to delete the active model */
   status = ml_service_model_delete (key, 1U);
-  EXPECT_EQ (ML_ERROR_NONE, status);
-
-  /* no active model */
-  status = ml_service_model_get_activated (key, &activated_model_info);
   EXPECT_EQ (ML_ERROR_INVALID_PARAMETER, status);
```

Comment:
### Question
In this case, which error code is much better `ML_ERROR_PERMISSION_DENIED` or `ML_ERROR_INVALID_PARAMETER`?